tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -12,9 +12,11 @@ Create container with port mappings
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start webserver
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10 --tries=10
```

Comment:
With --tries=10 removed, it'll go back to the default of 20, which still may not be enough time.  How about using the robot built-in to have a little time in between tries, we use something like this in several tests:

```
Wait Until Keyword Succeeds  20x  200 milliseconds ...
```